.github/workflows/release.yml

```diff
@@ -33,6 +33,10 @@ jobs:
         run: |
           ./gradlew -Pversion="${{ github.event.inputs.version }}" -PjavafxPlatform=linux --stacktrace :ice-adapter:shadowJar
 
+      - name: Build No-JavaFX Files
+        run: |
+          ./gradlew -Pversion="${{ github.event.inputs.version }}" -PjavafxPlatform=linux -PjavafxClasspath=compileOnly --stacktrace :ice-adapter:shadowJar
```

Comment:
Don't we technically need a windows and a linux version of this sonce the javafxPlatform is still used and part of the implementation?

Reply:
For non-java clients yes. (Pythpn, Eternal)